rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -136,6 +136,16 @@ public static boolean isReleaseTx(BtcTransaction tx, Federation federation, Brid
         return false;
     }
 
+    public static boolean isMigrationTx(BtcTransaction btcTx, Federation activeFederation, Federation retiringFederation, Context btcContext, BridgeConstants bridgeConstants) {
+        if (retiringFederation == null) {
+            return false;
+        }
+        boolean moveFromRetiring = isReleaseTx(btcTx, retiringFederation, bridgeConstants);
+        boolean moveToActive = isLockTx(btcTx, activeFederation, btcContext, bridgeConstants);
```

Comment:
isLockTx has not a really cool name reaching this point

It is supposed to answer the question if someone is trying to move btc into rsk, but that's not exactly what it is doing. Also the method is overloaded in a slightly ugly way. We should have method for p2pkh, p2sh, etc. and check for fed inputs / outputs where it's needed. But maybe in a future pr.

I think this name is error prone.